src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSupportsSharing/SharedMediaItem/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { Box, Text } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { ObjectStatusPicker } from 'components/ObjectStatusPicker';
+import { UploadingObject } from 'components/UploadManager';
+import { SharedLiveMedia, uploadState, Video } from 'types/tracks';
+import { Nullable } from 'utils/types';
+import { AllowedDownloadButton } from './AllowedDownloadButton';
+import { DisallowedDownloadButton } from './DisallowedDownloadButton';
+import { DeleteUploadButton } from './DeleteUploadButton';
+import { DownloadUploadLink } from './DownloadUploadLink';
+import { RetryButton } from './RetryButton';
+import { StartSharingButton } from './StartSharingButton';
+import { StopSharingButton } from './StopSharingButton';
+
+const messages = defineMessages({
+  noFilenameUploadFailed: {
+    defaultMessage: 'Upload has failed',
+    description:
+      "A message displayed when upload has failed, and so the file hasn't any filename.",
+    id: 'component.SharedMediaItem.noFilenameUploadFailed',
+  },
+  retryUploadFailedLabel: {
+    defaultMessage: 'Retry',
+    description:
+      'Label of the button allowing to retry upload when it has failed.',
+    id: 'component.SharedMediaItem.retryUploadFailedLabel',
+  },
+});
+
+interface SharedMediaItemProps {
+  isShared: Nullable<boolean>;
+  onRetryFailedUpload: (sharedMediaLiveId: string) => void;
+  sharedMedia: SharedLiveMedia;
+  uploadingObject?: UploadingObject;
+  video: Video;
+}
+
+export const SharedMediaItem = ({
+  isShared,
+  onRetryFailedUpload,
+  sharedMedia,
+  uploadingObject,
+  video,
+}: SharedMediaItemProps) => {
+  const intl = useIntl();
+  const sharedMediaTitle =
+    sharedMedia.title ||
+    uploadingObject?.file?.name ||
+    intl.formatMessage(messages.noFilenameUploadFailed);
+
+  return (
+    <Box
+      direction="row"
+      align="center"
+      fill="horizontal"
+      height="60px"
+      gap="medium"
+      pad={{ horizontal: 'small', vertical: 'small' }}
+    >
+      {sharedMedia.show_download ? (
+        <AllowedDownloadButton sharedLiveMediaId={sharedMedia.id} />
+      ) : (
+        <DisallowedDownloadButton sharedLiveMediaId={sharedMedia.id} />
+      )}
```

Comment:
To be honest I'm lost with the name you choose. You should choose a more relevant name to avoid confusion.